web/client-js/src/views/BoardView.vue

```diff
@@ -1,10 +1,29 @@
 <template>
   <div v-if="board" id="board">
-    <KanbanColumn
-      v-for="(column, $columnIndex) of board.getColumnList()"
-      :key="$columnIndex"
-      :column="column"
-    />
+    <div id="columns">
+      <div
+        v-for="(column, $columnIndex) of board.getColumnList()"
+        :key="$columnIndex"
+      >
```

Comment:
Can we please have the corresponding bracket of a tag on the same line as the last significant text?

This changeset illustrates the poor readability with the standalone `>`, `/>`, etc, bouncing all over the places.